app/views/projects/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @project.name%></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestant_count %></p>
+<p>Average Contestant Experience: <%= @project.contestant_experience_avg %></p>
+
+<h2>Add Contestant to Project (by id)</h2>
+<%= form_with url: "/projects/#{@project.id}", method: :patch, data: { turbo: false } do |f| %>
+  <%= f.label :id %>
+  <%= f.text_field :contestant_id %>
+
+  <%= f.submit 'Add Contestant' %>
+<% end %>
```

Comment:
See my comments in your `projects_controller` because we creating a new relationship between an existing contestant and an existing project, we will want to change our `url` and our `method` on this form. So something like:
```ruby
<%= form_with url: "/project/#{@project.id}/contestants", method: :post, local: true do |form| %>
  <%= form.label :contestant_id %>
  <%= form.number_field :contestant_id %>

  <%= form.submit "Add Contestant To Project" %>
<% end %>
```